.github/workflows/release.yaml

```diff
@@ -90,10 +91,14 @@ jobs:
           path: homebrew-tap
       - name: Update homebrew
         working-directory: homebrew-tap
+        env:
+          GORELEASER_ARTIFACTS: "${{ steps.goreleaser.outputs.artifacts }}"
```

Comment:
Example of this JSON found [here](https://github.com/OpsLevel/kubectl-opslevel/actions/runs/8054024555/job/21997675962) in the `release-brew` workflow, `Run GoReleaser` step, under `Artifacts output`.

This [github action's docs](https://github.com/goreleaser/goreleaser-action?tab=readme-ov-file#outputs) advertise this value as JSON.